apps/picsa-apps/dashboard/src/app/modules/crop-information/crop-information.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="page-content">
+  <div style="display: flex; align-items: center">
+    <h2 style="flex: 1">Crop Probabilities</h2>
+    <button mat-raised-button color="primary" (click)="navigateToEntryPage()">Add New Entry</button>
+  </div>
+  @if(service.cropProbabilities){
+  <table mat-table [dataSource]="service.cropProbabilities">
```

Comment:
nit(non-blocking)
I've added a shared `picsa-data-table` component to reduce the amount of boilerplate needed every time we create a new mat-table. Can you try import from `libs\shared\src\features\data-table\data-table.component.ts` and see if it works as a drop-in replacement for the table code?